x/concentrated-liquidity/pool.go

```diff
@@ -45,8 +45,8 @@ func (k Keeper) GetPool(ctx sdk.Context, poolId uint64) (poolmanagertypes.PoolI,
 	return poolI, nil
 }
 
-// getPoolById returns a concentratedPoolExtension that corresponds to the requested pool id. Returns error if pool id is not found.
-func (k Keeper) getPoolById(ctx sdk.Context, poolId uint64) (types.ConcentratedPoolExtension, error) {
+// GetPoolById returns a concentratedPoolExtension that corresponds to the requested pool id. Returns error if pool id is not found.
+func (k Keeper) GetPoolById(ctx sdk.Context, poolId uint64) (types.ConcentratedPoolExtension, error) {
```

Comment:
Per my earlier comment,
- Is it possible to use `GetPool` instead
- If not, can we rename this to `GetConcentratedPoolI` because semantically `GetPool` also gets pool by id so current names might start feeling confusing in the future

Reply:
Used GetPool and unexported getPoolById!

https://github.com/osmosis-labs/osmosis/pull/4051/commits/3b878d6a453dff3fdbbe471361d8381947aa8b17